### PR TITLE
Externalize local cache to `webext-storage-cache`

### PR DESCRIPTION
**Pull request by notlmn**
Externalizes caching to https://github.com/bfred-it/webext-storage-cache dependency.

Also adds caching to `clean-issue-filters` as promised in https://github.com/sindresorhus/refined-github/pull/2114#issuecomment-499763367.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
 		"shorten-repo-url": "^1.5.1",
 		"tiny-version-compare": "^1.0.0",
 		"type-fest": "^0.4.1",
-		"webext-detect-page": "^0.9.1",
 		"webext-domain-permission-toggle": "^0.1.0",
 		"webext-dynamic-content-scripts": "^5.0.2-0",
 		"webext-options-sync": "^0.15.5",
+		"webext-storage-cache": "0.0.0",
```
Comment (notlmn) [inline package.json]: Paging @bfred-it on this, can you publish another version like `0.0.1`.

**Comment by bfred-it** [comment]
Avoid adding/changing cache to many more features because I plan on changing the API of that module to make it look like it’s a plain memoized function call.

**Comment by notlmn** [comment]
> Avoid adding/changing cache to other features

Are you referring to adding caching to `clean-issue-filters` or modifying caching of existing features from `cache.ts` to `webext-storage-cache`?

**Comment by bfred-it** [comment]
Sorry I wasn’t clear; I mean *after this PR* don’t make other cache-related changes. This PR is ok

**Comment by sindresorhus** [comment]
Seems to work fine. Merge?

**Pull request merged**